### DISCUSSION TITLE
Welcome to 24-25-IdSw1-SDR Discussions!

### DISCUSSION THREAD

**Discussion by natiicruuz** [natiicruuz/24-25-IdSw1-SDR#1] [Announcements]
## 👋 ¡Bienvenido!
  Usamos Discusiones como un lugar para conectarnos con otros miembros de nuestra comunidad. Esperamos que usted:
  * Haga las preguntas que le interesen.
  * Compartir ideas.
  * Interactuar con otros miembros de la comunidad.
  * Dar la bienvenida a los demás y tener la mente abierta. Recuerde que esta es una comunidad que
  construir juntos 💪.

  Para comenzar, comenta a continuación con una presentación tuya y cuéntanos qué haces con esta comunidad.

**Comment by mmasias** [comment]
El [proyecto](https://github.com/users/natiicruuz/projects/1) mola!!! ¿Me dan acceso para poder dejar uno que otro comentario por allí? 🥹🥹🥹

**Reply by natiicruuz** [reply]
Hola! Acabo de enviarle el enlace de invitacion

**Reply by mmasias** [reply]
Recibido. Thanks!!!

**Comment by natiicruuz** [comment]
Hola estoy probando si respondiendo directamente desde el correo se pone en
el repo :)

Vamos a ver si sirve

*Natalia Cruz Babbar*

***@***.*** ***@***.***>*
*+34 637558211*

**Comment by mmasias** [comment]
Dudas de GitHub!!!!
